internal/controller/cephfscg/volumegroupsourcehandler.go

```diff
@@ -30,6 +30,8 @@ var (
 	RestorePVCinCGNameFormat      = "vs-cg-%s"
 	SnapshotGroup                 = "snapshot.storage.k8s.io"
 	SnapshotGroupKind             = "VolumeSnapshot"
+
+	createdByRamenLabel = "ramendr.openshift.io/resource-created-by-ramen"
```

Comment:
Move this to internal/controller/core/labels.go.

1. also add a function in the same file addRamenLabels(labels map[string]string) map[string]string. You can call this function from every file in this PR than modifying the label directly.